### PR TITLE
Changing the caption for the GithubTeacher slide

### PR DESCRIPTION
**Pull request by unikornintech**
Please review my birthday slide

Tell ME Happy Belated Birthday!!

### CODE REVIEW

**Comment by JannaHawkins** [comment]
👍

File: _posts/0000-01-06-unikornintech.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Unikornintech's slides!"
 ---
 
 ![panda-birthday](https://cloud.githubusercontent.com/assets/16547949/25400994/60f64804-29c2-11e7-8a6d-4401d7d294b9.jpg)
 {: .center}
 
-Caption here
+Happy Belated Birthday Unikornintech<br>APRIL 20TH<br>ONLY THE BEST DATE EVER!
```
Comment (JannaHawkins) [inline _posts/0000-01-06-unikornintech.md:9]: Happy Birthday!!!

**Review by JannaHawkins — COMMENTED** [review]
@unikornintech Happy Birthday

File: _posts/0000-01-06-unikornintech.md (hunk 1 above)
Comment (unikornintech) [reply _posts/0000-01-06-unikornintech.md:9]: Thank You!! 😄 😊

**Review by beardofedu — APPROVED** [review]

**Pull request merged**